screen/model/media/videohandler.js

```diff
@@ -29,4 +35,15 @@ module.exports = class VideoHandler extends MediaHandler {
         this.videoNode.onended = (e) => this.destroy();
     }
 
+    /* Either plays or pauses video, depending on the given argument */
+    play(play = true){
```

Comment:
Will this be used in more places later? Otherwise I think it would be clearer to put this.videoNode.play() (or pause) directly in handleMessage, even with the surrounding if-statement.

Reply:
I guess it might be good to have separate from the message handling if we want some custom video handler for a specific purpose (e,g. something that autogenerates video from code?) and want to override the play method :)

Reply:
Fair enough, it's a bit confusing that the method is called "play" though. Make two methods?